smoke_tests_wsgi.sh

```diff
@@ -3,5 +3,7 @@ set -ex
 
 source venv/bin/activate
 
+until echo > /dev/tcp/localhost/9000; do sleep 1; done
```

Comment:
`localhost` is hardcoded but it's fine as this operation is local to this container